build/platform_and_feature_flags.props

```diff
@@ -6,19 +6,19 @@
     <DefineConstants>$(DefineConstants);NET5_WIN;SUPPORTS_CONFIDENTIAL_CLIENT;SUPPORTS_CUSTOM_CACHE;SUPPORTS_OS_SYSTEM_BROWSER;SUPPORTS_BROKER;SUPPORTS_WEBVIEW2</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkUap)' ">
-    <DefineConstants>$(DefineConstants);WINDOWS_APP;SUPPORTS_BROKER;SUPPORTS_CUSTOM_CACHE</DefineConstants>
+    <DefineConstants>$(DefineConstants);WINDOWS_APP;SUPPORTS_BROKER;SUPPORTS_OS_SYSTEM_BROWSER;SUPPORTS_CUSTOM_CACHE</DefineConstants>
```

Comment:
UAP / UWP (Universal Windows Platform) does not support SYSTEM_BROWSER

Reply:
Updated to removed SUPPORTS_OS_SYSTEM_BROWSER from all configurations